ioctl/newcmd/node/nodedelegate.go

```diff
@@ -88,7 +96,8 @@ func NewNodeDelegateCmd(c ioctl.Client) *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
 			var err error
```

Comment:
`err` is not used seemly.

Reply:
used in here
https://github.com/iotexproject/iotex-core/blob/56c8c09f05e58ad4b2469f39912eccab3bd368da/ioctl/newcmd/node/nodedelegate.go#L234-L236